mne/decoding/tests/test_transformer.py

```diff
@@ -160,3 +160,27 @@ def test_epochs_vectorizer():
     # Test init exception
     assert_raises(ValueError, vector.fit, epochs, y)
     assert_raises(ValueError, vector.transform, epochs, y)
+
+
+def test_vectorizer():
+    """Test Vectorizer."""
+    data = np.random.rand(150, 18, 6)
+    vect = Vectorizer()
+    result = vect.fit_transform(data)
+    assert_equal(result.ndim, 2)
+    # check inverse_trasnform
+
+    orig_data = vect.inverse_transform(result)
+    assert_equal(orig_data.ndim, 3)
+    assert_array_equal(orig_data, data)
```

Comment:
it does not seem you test inverse_transform with a different number of samples. this should pass. You could do:

assert_array_equal(vect.inverse_transform(result[1:]), data[1:])

ok?